registry/help.md

```diff
@@ -6,12 +6,13 @@ title: Get help
 
 If you need help, or just want to chat, you can reach us:
 
-- on irc: `#docker-distribution` on freenode
-- on the [mailing list](https://groups.google.com/a/dockerproject.org/forum/#!forum/distribution) (mail at <distribution@dockerproject.org>)
+- on the [Docker forums](https://forums.docker.com/c/open-source-projects/opensrcreg).
+- on irc: `#docker-distribution` on freenode.
```

Comment:
Is IRC still in active use for distribution? Otherwise we may want to refer to the Docker community slack.

Reply:
I have no idea. Do you know how should be looped in to confirm that?

Reply:
Let me ask @stevvooe and @dmcgowan

Reply:
Yup, replacing IRC with Community slack should be fine.

Reply:
Also; they are not actively monitoring the forums, so we should be careful referring people there

Reply:
Even if they don't monitor the forums, it's still a good place for people to ask for help. Sometimes you're lucky and find that someone had the some problem and was able to fix it.

Reply:
It's fine to mention it for that reason; just thought I'd mention it, in case the expectations were that the distribution maintainers are very active there 😅